package.json

```diff
@@ -43,6 +43,7 @@
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.18.2",
     "express": "^4.16.2",
+    "express-jwt": "^5.3.1",
```

Comment:
You aren't using this package.

Reply:
Yes, I added it prematurely, it should be on the next pr